src/sonic-yang-models/tests/yang_model_tests/yangModelTesting.py

```diff
@@ -202,11 +216,22 @@ def logStartTest(self, desc):
     """
         Load Config Data and return Exception as String
     """
-    def loadConfigData(self, jInput):
+    def loadConfigData(self, jInput, verify=None):
         s = ""
```

Comment:
Could you add comment about the type and description of "verify"?

Reply:
Sure, Let me add a description here.

Reply:
done

Reply:
Good